tooling/v8-snapshot/README.md

```diff
@@ -0,0 +1,185 @@
+    ".. many more ..",
+    "./node_modules/send/index.js",
+    "./node_modules/send/node_modules/http-errors/index.js"
+  ],
+  "healthy": [
+    "./node_modules/accepts/index.js",
+    "./node_modules/array-flatten/array-flatten.js",
+    "./node_modules/body-parser/lib/read.js",
+    "./node_modules/body-parser/lib/types/json.js",
+    ".. many more ..",
+    "./node_modules/unpipe/index.js",
+    "./node_modules/utils-merge/index.js",
+    "./node_modules/vary/index.js",
+    "./snapshot/snapshot.js"
+  ],
+  "deferredHashFile": "yarn.lock",
+  "deferredHash": "216a61df3760151255ce41db2a279ab4d83b4cb053687766b65b19c4010753a2"
+}
+```
+
+As you can see this meta file can be used as is as long as the content of the `yarn.lock` file
+doesn't change and in that case the doctor step does not have to be repeated.
+
+Another option is to provide the meta information to the doctor via `previousDeferred`,
+`previousHealthy`, etc. in order to have it use this information instead of starting totally
+from scratch.
+
+Aside from this script the snapshot generator allows making and installing the snapshot into
+our app. See [makeAndInstallSnapshot][makeAndInstallSnapshot].
+
+## Env Vars
+
+- `SNAPSHOT_BUNDLER` overrides Go binary to create the JavaScript bundle used to snapshot
+- `SNAPSHOT_KEEP_CONFIG` when set will not delete the temporary JSON config file that is
+	provided to the snapshot bundler
+ 
+[doctor-next-stage]:https://github.com/cypress-io/cypress/blob/bbabd12c9aae3d3eef0cb077d3e8628a53eea623/tooling/v8-snapshot/src/doctor/snapshot-doctor.ts#L628
+[doctor-class]:https://github.com/cypress-io/cypress/blob/bbabd12c9aae3d3eef0cb077d3e8628a53eea623/tooling/v8-snapshot/src/doctor/snapshot-doctor.ts#L261
+[makeAndInstallSnapshot]:https://github.com/cypress-io/cypress/blob/bbabd12c9aae3d3eef0cb077d3e8628a53eea623/tooling/v8-snapshot/src/snapshot-generator.ts#L669
+
+[blueprint-config]:https://github.com/cypress-io/cypress/blob/bbabd12c9aae3d3eef0cb077d3e8628a53eea623/tooling/v8-snapshot/src/blueprint.ts#L51
+[globals-strict-code]:https://github.com/cypress-io/cypress/blob/bbabd12c9aae3d3eef0cb077d3e8628a53eea623/tooling/v8-snapshot/src/blueprint/globals-strict.js
+
+[generation-opts]:https://github.com/cypress-io/cypress/blob/bbabd12c9aae3d3eef0cb077d3e8628a53eea623/tooling/v8-snapshot/src/snapshot-generator.ts#L112
+[resolver-map]:https://github.com/cypress-io/cypress/blob/bbabd12c9aae3d3eef0cb077d3e8628a53eea623/tooling/v8-snapshot/src/snapshot-generator.ts#L126
+[snapshot-verifier]:https://github.com/cypress-io/cypress/blob/bbabd12c9aae3d3eef0cb077d3e8628a53eea623/tooling/v8-snapshot/src/snapshot-verifier.ts#L10
+
+[snapshot-doctor]:https://github.com/cypress-io/cypress/blob/bbabd12c9aae3d3eef0cb077d3e8628a53eea623/tooling/v8-snapshot/src/doctor/snapshot-doctor.ts
+[snapshot-doctor-class]:https://github.com/cypress-io/cypress/blob/bbabd12c9aae3d3eef0cb077d3e8628a53eea623/tooling/v8-snapshot/src/doctor/snapshot-doctor.ts#L261
```

Comment:
nit: duplicate of `[doctor-class]` link above

Reply:
Good catch. Updated here: [`3accc37` (#23566)](https://github.com/cypress-io/cypress/pull/23566/commits/3accc371181a876d646b3c3cf5c35dd60440be3b)